### DISCUSSION TITLE
How to add dynamic attribute to the component

### DISCUSSION THREAD

**Discussion by dannyphang** [heroui-inc/heroui#875] [Help] [answered]
![image](https://user-images.githubusercontent.com/68151810/201540068-9a67752d-46c8-4272-916b-a3516c9df4d1.png)
![image](https://user-images.githubusercontent.com/68151810/201540214-63f59dad-aee3-4693-9904-4a15e126c2d2.png)

![image](https://user-images.githubusercontent.com/68151810/201540164-618508f0-bf43-4e90-bdcc-eecdcec43764.png)

how do I add a new/change an attribute inside a function?

**Comment by tianenpang** [comment, marked as answer]
Hi @dannyphang you can use it with [`useInput`](https://nextui.org/docs/components/input#helper-text), here's an example [codesandbox.io/s/nextui-discussion-875-mm17f1](https://codesandbox.io/s/nextui-discussion-875-mm17f1?file=%2Fsrc%2Fapp.js).
